home/static/home/CSS/about.css

```diff
@@ -0,0 +1,27 @@
+.main-bg {
+    text-align: center;
+}
```

Comment:
I see you didn't put in "main-bg" class the background color.
what is the background color of the new about page is then?

Reply:
It's an extend of home/base.html so the color is:
```
.main-bg{
	background-color: #d9b6a3;
}
```
Is this color ok or should I change it?